shell/src/main/java/alluxio/cli/fs/command/SetReplicationCommand.java

```diff
@@ -96,29 +98,31 @@ private void setReplication(AlluxioURI path, Integer replicationMax, Integer rep
   public int run(CommandLine cl) throws AlluxioException, IOException {
     String[] args = cl.getArgs();
     AlluxioURI path = new AlluxioURI(args[0]);
-    Integer replicationMax = cl.hasOption("max") ? Integer.valueOf(cl.getOptionValue("max")) : null;
-    Integer replicationMin = cl.hasOption("min") ? Integer.valueOf(cl.getOptionValue("min")) : null;
-    boolean recursive = cl.hasOption("R");
+    Integer replicationMax = cl.hasOption(MAX_OPTION.getLongOpt())
```

Comment:
I added a function not too long ago that helps remove some of this code duplication for Integer arguments in shell commands.

https://github.com/Alluxio/alluxio/blob/master/shell/src/main/java/alluxio/cli/fs/FileSystemShellUtils.java#L231-L238

Up to you if you want to use it.

Reply:
seems not applicable here as they need to be nullable for this case to indicate value not set